tests/Unit/Application/Queries/GetShoppingListDetail/GetShoppingListDetailQueryTest.php

```diff
@@ -55,7 +56,7 @@ public function test(): void
             new ShoppingItemDetailModel(2, 'Bananas', false),
         ]);
 
-        $actual = $this->query->execute('my-groceries');
+        $actual = $this->query->execute(new GetShoppingListDetailRequest('my-groceries', 'Bananas'));
```

Comment:
Suggestion: this is the correct update to this test, i.e. get it using the request class. My only suggestion is you change the string 'Bananas' because that isn't a valid filter variable... i.e. the string makes the test a bit unrealistic.